sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,203 @@
+| yes      | `11111111111mdWK2VXcrA1e7dnvidC` | `SP00000000001WPAWSDEDMQ0B9J72P0KAK2` | `yes-sip-28` | `000000000000000000007965732d7369702d3238` | `OP_DUP` `OP_HASH160` `000000000000000000007965732d7369702d3238` `OP_EQUALVERIFY` `OP_CHECKSIG` |
+| no       | `111111111111ACW5wa4RwyeKYEAzMD` | `SP000000000006WVSDEDMQ0B9J73E2TN78`  | `no-sip-28`  | `00000000000000000000006e6f2d7369702d3238` | `OP_DUP` `OP_HASH160` `00000000000000000000006e6f2d7369702d3238` `OP_EQUALVERIFY` `OP_CHECKSIG` |
+
+The addresses have been generated as follows:
+
+- Encode `<message>` in ASCII, with 0-padding.
+- Use the resulting `<encoding>` in the Bitcoin script`OP_DUP` `OP_HASH160` `<encoding>` `OP_EQUALVERIFY` `OP_CHECKSIG`.
+- The Bitcoin address is the `base58check` of the hash of the Bitcoin script above.
+- The Stacks address is the `c32check-encoded` Bitcoin address.
+
+Stackers (pool and solo) vote by sending Stacks dust to the corresponding Stacks address from the account where their Stacks are locked.
+
+Solo stackers only can also vote by sending a bitcoin dust transaction (6000 sats) to the corresponding bitcoin address.
+
+#### For Non-Stackers:
+
+Users with liquid STX can vote on proposals using the Ecosystem DAO. Liquid STX is the user’s balance, less any STX they have locked in the PoX stacking protocol, at the block height at which the voting started (preventing the same STX from being transferred between accounts and used to effectively double vote). This is referred to generally as "snapshot" voting.
+
+For this SIP to pass, 66% of all liquid STX committed by voting must be in favor of the proposal. This precedent was set by [SIP-015](https://github.com/stacksgov/sips/blob/main/sips/sip-015/sip-015-network-upgrade.md).
+
+The act of not voting is the act of siding with the outcome, whatever it may be. We believe that these thresholds are sufficient to demonstrate interest from Stackers -- Stacks users who have a long-term interest in the Stacks blockchain's successful operation -- in performing this upgrade.
+
+## Appendix
+[1] https://github.com/stacks-network/sbtc/issues/52
+
+[2] https://github.com/stacks-network/sbtc/pull/186
+
+### Specification
+
+#### Deposits
+
+The main steps of the sBTC Deposit flow will be as follows:
+
+1. **Deposit request:** A bitcoin holder creates a transaction on Bitcoin.
+2. The deposit transaction contains a UTXO (deposit UTXO) spendable by sBTC Signers, with an OP_DROP payload.
+3. The payload contains the recipient address of the sBTC among other relevant info for the deposit.
+   - The relevant info could contain a fee suggestion or max_fee.
+4. **Proof of deposit:** The bitcoin holder submits a proof of deposit on Stacks by invoking the Deposit API.
+5. **Deposit accept:** 
```

Comment:
missing details here

Reply:
@andrerserrano @aldur

Reply:
Can you elaborate - which details would you like to see? The SIP is on the signer criteria, not on the technical details of the deposit flow. We aimed to provide a high level overview of the system without going into too many tech details.

Reply:
`5. **Deposit accept:** `

has no definition like the other items do

Reply:
Got it. That was a formatting issue, these were meant to be sub-bullets. I've updated this - thanks!